packages/server/lib/hooks/hooks.ts

```diff
@@ -353,15 +356,8 @@ export async function credentialsTest({
             if (response.status && response.status >= 200 && response.status < 300) {
                 return Ok({ logs, tested: true });
             }
-
-            logs.push({ type: 'log', level: 'error', message: `Failed verification for endpoint: ${endpoint}`, createdAt: new Date().toISOString() });
-        } catch (err) {
-            logs.push({
-                type: 'log',
-                level: 'error',
-                message: `Error testing endpoint: ${endpoint},  ${stringifyError(err)}`,
-                createdAt: new Date().toISOString()
-            });
+        } catch {
+            // Already covered
```

Comment:
not sure I follow

Reply:
historically the proxy calls were throwing the last error allowing us to log the last error, but we already log those errors, it's just throwing to allow the SDK to bubble up the request but it's useless in this case